### PR TITLE
add_triage_signatures: format SIGNATURES

### PR DESCRIPTION
**Pull request by vrutkovs**
Format signature class list to use one class per line to avoid PR rebases when a new class is added

### CODE REVIEW

File: tools/add_triage_signature.py [hunk 1]
```diff
@@ -1248,11 +1248,24 @@ def _update_ticket(self, url, issue_key, should_update=False):
...
+    HostsExtraDetailSignature,
+    StorageDetailSignature,
+    InstallationDiskFIOSignature,
+    LibvirtRebootFlagSignature,
+    MediaDisconnectionSignature,
+    ConsoleTimeoutSignature,
+    AgentStepFailureSignature,
+    CNIConfigurationError,
+    MustGatherAnalysis,
+    OSInstallationTime
```
Comment (romfreiman) [inline tools/add_triage_signature.py:1267]: ,

Comment (vrutkovs) [reply tools/add_triage_signature.py]: Fixed and rebased

**Review by mkowalski — APPROVED** [review]

**Comment by omertuc** [comment]
Let's close this in favor of https://github.com/openshift-assisted/assisted-installer-deployment/pull/247 ?

**Pull request closed without merging**